<launch>
  <!-- <arg name="network_interface" default="enx705dccfe7579"/> -->
  <!-- <arg name="network_interface" default="enx00e04cca5466"/>  DOCKHUB -->
  <arg name="network_interface" default="enx0c37961999e8"/> <!--  AERO -->

  <node name="rft_interface_node" pkg="rft64_6a01_ros_interface" type="interface.py" output="screen" args="$(arg network_interface)"/>

  <node name="rft_rqt_plot" pkg="rqt_gui" type="rqt_gui" args=" --perspective-file $(find rft64_6a01_ros_interface)/rqt/rft_xyz_rpy.perspective"/>
</launch>